src/lib/components/streams/channel-messages.svelte

```diff
@@ -66,6 +66,7 @@
             <div class="mb-4">
                 <div class="text-secondary">Link</div>
                 <div class="text-break">{msg?.link}</div>
+                <h6>{JSON.stringify(msg.log)}</h6>
```

Comment:
whats that?

Reply:
That was for testing and I forgot to remove it. I will remove it.